packages/syft/src/syft/core/tensor/lazy_repeat_array.py

```diff
@@ -62,8 +65,12 @@ def __add__(self, other: Any) -> lazyrepeatarray:
 
         if self.data.shape == other.data.shape:
             return self.__class__(data=self.data + other.data, shape=self.shape)
+        else:
+            # Rasswanth : Fix after addition of integration tests.
+            print("Lazy Repeat adding with mismatched shapes")
```

Comment:
Q: Can't we use the `logger` here? (maybe add the `info`)

Reply:
This I will plan to remove before merge,I did this quick fix for a demo.